### ISSUE TITLE
Limit accessibility check to WCAG failures, not best practices

### ISSUE THREAD

**Issue by aardrian** [sonarwhal/sonarwhal#1087]
When running an online scan at sonarwhal.com, the results from aXe are presented as accessibility failures, but it is not clear what is a true WCAG violation and what is a best practice as defined by the aXe team.

By default, limit the output of Sonarwhal to only present WCAG failures, or flag items that are best practices as such. I understand that the results from aXe expose whether a failure is a best practice or not. The aXe browser plug-in makes that distinction, so aligning with the aXe browser plug-in would also be a bonus.

Alternatively, an option to categorize failures (or disable/enable) by WCAG level and/or best practice might be a nice feature for those reading these reports.

## Example

If I run [Delta.com through Sonarwhal](https://sonarwhal.com/scanner/1879c43d-9a2a-47b3-8ed1-d0ead83792ac) (specifically the [system unavailable](https://www.delta.com/content/www/en_US/system-unavailable1.html) page), the report returns the following messages (among others):

> ERROR
> Page must contain one main landmark
> ERROR
> Content should be contained in a landmark region

If I run aXe in my browser, the report excludes the first error and says (note use of _should_ and _best-practice_):

> Content should be contained in a landmark region
> Issue tags: category: keyboard best-practice

Neither of the issues listed in the online scanner is a WCAG failure, but the _must_ language can confuse users who do not know WCAG and are using the tool to prioritize remediation efforts.

## Environment

Sonarwhal.com online scanner

## URLs in this report

* __Sonarwhal:__ [sonarwhal.com/scanner/1879c43d-9a2a-47b3-8ed1-d0ead83792ac](https://sonarwhal.com/scanner/1879c43d-9a2a-47b3-8ed1-d0ead83792ac)
* __Original Page:__ [delta.com/content/www/en_US/system-unavailable1.html](https://www.delta.com/content/www/en_US/system-unavailable1.html)

**Comment by molant** [comment]
Looks like there is a problem in aXe's documentation. I've opened [this issue](https://github.com/dequelabs/axe-core/issues/907) in their repo.

Our intention was to only run WCAG 2.0 Level A and AA so I'll update the rule to be explicit on that aspect.

Thanks for reporting this @aardrian!

**Comment by molant** [comment]
There's a [PR now](https://github.com/sonarwhal/sonarwhal/pull/1090)

**Comment by aardrian** [comment]
I just ran Sonarwhal against the same Delta page, and it now comes back without any errors from aXe. Since I last ran it, the Delta page has added a `lang` attribute and a `<main>` element, so maybe I chose a poor example. I mean, who thought I would catch them the day before they deployed fixes?
